ec3730.xcodeproj/project.pbxproj

```diff
@@ -11,13 +11,15 @@
 		2E07167528FEB055009C3844 /* WhoIsXmlContactsResult.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2E07167428FEB055009C3844 /* WhoIsXmlContactsResult.swift */; };
 		2E07167F28FEB837009C3844 /* WhoIsXmlContactsSectionModel.json in Resources */ = {isa = PBXBuildFile; fileRef = 2E07167E28FEB837009C3844 /* WhoIsXmlContactsSectionModel.json */; };
 		2E07168128FEC3F3009C3844 /* WhoisXmlContactsService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2E07168028FEC3F3009C3844 /* WhoisXmlContactsService.swift */; };
-		846CDA83291CF52C00DDB5AA /* WhoIsXmlGeoLocationSectionModel.json in Resources */ = {isa = PBXBuildFile; fileRef = 846CDA82291CF52C00DDB5AA /* WhoIsXmlGeoLocationSectionModel.json */; };
 		846CDA85291CF6A800DDB5AA /* ApiKeys.swift in Sources */ = {isa = PBXBuildFile; fileRef = 846CDA84291CF6A800DDB5AA /* ApiKeys.swift */; };
 		846CDA87291CF84400DDB5AA /* WhoIsXmlGeoLocationSectionModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 846CDA86291CF84400DDB5AA /* WhoIsXmlGeoLocationSectionModel.swift */; };
 		846CDA89291CF87300DDB5AA /* WhoIsXmlGeoLoactionService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 846CDA88291CF87300DDB5AA /* WhoIsXmlGeoLoactionService.swift */; };
 		846CDA8B291CF88B00DDB5AA /* WhoIsXmlGeoLocationResult.swift in Sources */ = {isa = PBXBuildFile; fileRef = 846CDA8A291CF88B00DDB5AA /* WhoIsXmlGeoLocationResult.swift */; };
+		84700BC729640F2F00113314 /* (null) in Resources */ = {isa = PBXBuildFile; };
```

Comment:
Why did this file change?

Reply:
This totally depends upon XCode execution, I didn't make any changes

Reply:
Please only include the relevant changes

Reply:
Sure